src/controller/CommissioningDelegate.h

```diff
@@ -679,6 +679,16 @@ struct GeneralCommissioningInfo
     ;
 };
 
+struct ICDManagementClusterInfo
+{
+    bool isLIT                  = false;
+    bool checkInProtocolSupport = false;
+
+    BitMask<app::Clusters::IcdManagement::UserActiveModeTriggerBitmap> userActiveModeTriggerHint =
+        BitMask<app::Clusters::IcdManagement::UserActiveModeTriggerBitmap>();
+    CharSpan userActiveModeTriggerInstruction = CharSpan();
```

Comment:
If we're just default-constructing, why do we need to list the initializers at all?

Reply:
updated, thanks